lib/components/DraftailEditor.js

```diff
@@ -857,7 +863,7 @@ class DraftailEditor extends Component<Props, State> {
     return (
```

Comment:
There is a bit of code above this that sets the toolbars’ `readOnly` – I think it should also be switched to:

```js
readOnly: isReadOnly
```

Reply:
Isn't this the one below?

```
 className={`Draftail-Editor${
          readOnly ? " Draftail-Editor--readonly" : ""	          isReadOnly ? " Draftail-Editor--readonly" : ""
        }${hidePlaceholder ? " Draftail-Editor--hide-placeholder" : ""}${	        }${hidePlaceholder ? " Draftail-Editor--hide-placeholder" : ""}${
          hasFocus ? " Draftail-Editor--focus" : ""	          hasFocus ? " Draftail-Editor--focus" : ""
        }`}	        }`}
```

Reply:
Fixed.